### Вопросы

1. Что выведется в консоль? Переменная `unknownVariable` не определена

```javascript
  try {
    console.log("1");

    unknownVariable.push("123");

    console.log("2");
  } catch(err) {
    console.error("Возникла ошибка!", err);
  }
```

2. Сработает ли `catch`, если функция внутри таймаута содержит ошибку?

```javascript
  try {
    setTimeout(function() {
      noSuchVariable; // скрипт упадёт тут
    }, 1000);
  } catch (e) {
    console.error("Получена ошибка");
  }
```
